timesketch/api/v1/resources/event.py

```diff
@@ -1208,3 +1248,156 @@ def post(self, sketch_id):
         )
 
         return HTTP_STATUS_CODE_OK
+
+
+class EventTagResource(resources.ResourceMixin, Resource):
+    """Resource to add / remove a tag to an event."""
+
+    # The maximum number of events to tag in a single request.
+    MAX_EVENTS_TO_TAG = 500
+
+    # The maximum number of tags in a single request.
+    MAX_TAGS_PER_REQUEST = 500
+
+    @login_required
+    def post(self, sketch_id, event_id):
```

Comment:
Happy to remove it again, but IMHO it would be nice for a user that this is not yet implemented and what to use in the meantime

Reply:
ok this method is gone now